web/src/Setting.js

```diff
@@ -532,7 +536,8 @@ export function getProviderLogoURL(provider) {
     }
     return `${StaticBaseUrl}/img/social_${provider.type.toLowerCase()}.png`;
   } else {
-    return OtherProviderInfo[provider.category][provider.type].logo;
+    const info = OtherProviderInfo[provider.category][provider.type];
+    if (info) {return info.logo;}
```

Comment:
@qianxi0410 why the linter doesn't report this?

Reply:
@kofj you already added new item to `OtherProviderInfo`, why add `if` here?

Reply:
In other cases maybe cause page crash, so we need check it first.

Reply:
same as above